targets/TARGET_TOSHIBA/TARGET_TMPM46B/us_ticker.c

```diff
@@ -0,0 +1,138 @@
+/* mbed Microcontroller Library
+ * (C)Copyright TOSHIBA ELECTRONIC DEVICES & STORAGE CORPORATION 2017 All rights reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "us_ticker_api.h"
+#include "tmpm46b_tmrb.h"
+
+#define TMR16A_100US          0xFFFF
+#define TMRB_CLK_DIV          0x3
+
+static uint8_t us_ticker_inited = 0;                // Is ticker initialized yet?
+static volatile uint32_t us_ticker = 0;             // timer counter
+
+void INTTB1_IRQHandler(void)
+{
+    // This function is going to disable the interrupts if there are
+    us_ticker_irq_handler();
+}
+
+const ticker_info_t* us_ticker_get_info()
+{
+    static const ticker_info_t info = {
+			1875000,//1875000,
+            16     // 16 bit counter
+    };
+    return &info;
+}
+
+// initialize us_ticker
+void us_ticker_init(void)
```

Comment:
I've noticed in the init interrupts for HAL. you are not using NVIC_SetVector - dynamic vectors?

See https://docs.mbed.com/docs/mbed-os-handbook/en/5.2/advanced/porting_guide/ (Bootstrap section)

Reply:
@0xc0170, Added NVIC_SetVector - dynamic vectors